docker-entrypoint.sh

```diff
@@ -55,6 +55,12 @@ EOF
   mv /tmp/broker-merge.xml $CONFIG_PATH/broker.xml
 fi
 
+if [[ "$JOLOKIA_ALLOW_ORIGIN" ]]; then
+  cp $CONFIG_PATH/jolokia-access.xml /tmp/jolokia-access.xml
```

Comment:
Is there really a reason to use `jolokia-cors.xslt` file? The same effect can be achieved with
```
xmlstarlet ed --inplace -u  '/restrict/cors/allow-origin' -v '*' "$BROKER_HOME/etc/jolokia-access.xml"
```
and it's arguably clearer.

Reply:
That would be an improvement

Reply:
This still does the trick, I'm merging @dwickern would you mind to create another PR with the improvement?